### PR TITLE
PT-1115 SP8 patients are not displayed with correct time

### PR DESCRIPTION
**Pull request by franck-fourel**
use cbg instead of upload to determine the default display timezone, it will default to browser timezone

### CODE REVIEW

File: test/unit/utils/utils.test.js [hunk 1]
```diff
@@ -395,10 +395,11 @@ describe('utils', () => {
   });
 
   describe('getTimezoneForDataProcessing', () => {
+    // temporary change until we have consitent upload data
```
Comment (BigFab) [inline test/unit/utils/utils.test.js]:
```suggestion
    // temporary change until we have consistent upload data
```

File: app/core/utils.js [hunk 2]
```diff
@@ -301,7 +301,8 @@ utils.roundBgTarget = (value, units) => {
 
 utils.getTimezoneForDataProcessing = (data, queryParams) => {
   var timePrefsForTideline;
-  var mostRecentUpload = _.sortBy(_.filter(data, {type: 'upload'}), (d) => Date.parse(d.time)).reverse()[0];
+  // workaround a bug in upload timezone
+  var mostRecentUpload = _.sortBy(_.filter(data, {type: 'cbg'}), (d) => Date.parse(d.time)).reverse()[0];
```
Comment (yvon-dblg) [inline app/core/utils.js]: Does this change lead to a slower loading ?
There is far more cbg data than upload ones.

Comment (franck-fourel) [reply app/core/utils.js]: good point 
we could even bypass it as we don't have any timezone in our objects. Let me switch to a food object

**Comment by franck-fourel** [comment]
Validated on preview with sp8 user
before 
image
 after 
image

**Pull request merged**